### DISCUSSION TITLE
we should make 4 new mascots that go with the bark dog like in scratch 2.0

### DISCUSSION THREAD

**Discussion by rubiidev18alt** [cytrinc/bark-coding#157] [Admin General]
title, inspiration can be from https://scratch.mit.edu/projects/702737319/ but with much more unique ideas
edit: I also want to redesign the bark dog a Lil, we wouldn't wanna have copyright issues **right?**

**Comment by raynepaws** [comment]
i would like to propose a redesign of the bark sprite

**Reply by rubiidev18alt** [reply]
ah, OK! send me WHT it would look like.

**Reply by raynepaws** [reply]
i tried to keep it very minimalistic, since mascots are more memorable that way. if you would like anything added/changed just say so (this is a 9 minute sketch so nothing spectacular)
![bark dog proposed redesign](https://github.com/mariocraft987/bark-coding/assets/129226914/89d5d87a-53c6-4d43-a6db-6c2911aaa68f)

**Reply by rubiidev18alt** [reply]
this looks nice! suggestions:
for the site: make the theme as cozy as the mascot if you add it
there should be some more white parts on the paws and ears.
make a 2nd costume with a >;3 face and him doing a handstand because why not.
redo the belly circle thingy
make the proportions a bit shorter

**Reply by raynepaws** [reply]
thanks for the feedback :)

**Reply by mariocraft987** [reply]
Looks good! Too good!
